biolink-model.yaml

```diff
@@ -3889,7 +3937,7 @@ slots:
     is_a: participates in
     description: >-
       holds between a physical entity and a process, where the physical entity executes the process
```

Comment:
Would be good to update the description too :)

Reply:
reverted back to physical entity - then will change to chemical entity (which is now the parent of Nucleic Acid Entity) in the chemical branch refactor